utils/build.ps1

```diff
@@ -1601,15 +1634,15 @@ function Build-ASN1($Arch) {
 function Build-Certificates($Arch) {
   Build-CMakeProject `
     -Src $SourceCache\swift-certificates `
-    -Bin $BinaryCache\11 `
+    -Bin (Get-HostProjectBinaryCache Certificates) `
     -Arch $Arch `
     -UseBuiltCompilers Swift `
     -SwiftSDK $SDKInstallRoot `
     -BuildTargets default `
     -Defines @{
       BUILD_SHARED_LIBS = "NO";
-      SwiftASN1_DIR = "$BinaryCache\10\cmake\modules";
-      SwiftCrypto_DIR = "$BinaryCache\8\cmake\modules";
+      SwiftCrypto_DIR = (Get-HostProjectCMakeModules Crypto);
```

Comment:
Parens are probably not necessary on the RHS of an assignment.

Reply:
Oh, it shouldn't confuse things to have `Get-HostProjectCMakeModules Crypto;`? I definitely would prefer to remove the parenthesis then.

Reply:
Seems that they are needed :( - maybe you can try to clean them up subsequently? I'd love to get that noise removed.